pkg/tfgen/generate_schema_test.go

```diff
@@ -449,6 +449,16 @@ func TestRequiredInputWithDefault(t *testing.T) {
 	bridgetesting.AssertEqualsJSONFile(t, "test_data/required-input-with-default-schema.json", schema)
 }
 
+func TestNestedFullyComputed(t *testing.T) {
+	t.Parallel()
+	provider := testprovider.ProviderNestedFullyComputed()
+	schema, err := GenerateSchema(provider, diag.DefaultSink(io.Discard, io.Discard, diag.FormatOptions{
+		Color: colors.Never,
+	}))
+	require.NoError(t, err)
+	bridgetesting.AssertEqualsJSONFile(t, "test_data/nested-fully-computed-schema.json", schema)
```

Comment:
Same comment, inline the provider and make a precise assert, then the test will become readable